src/rust/ide/view/src/documentation.rs

```diff
@@ -154,6 +152,22 @@ impl Model {
         }
     }
 
+    /// Fixes indentation in the first line of documentation by looking at next line, as documentation
+    /// doesn't know what was its indentation in the Enso source file.
+    fn fix_indent_in_first_line(string: &str) -> String {
+        let mut processed = string.to_string();
+        let lines = processed.lines().collect::<Vec<&str>>();
```

Comment:
You can do that without allocating a vector. The most smart way I made out is:
```rust
let lines = processed.lines().take(2).collect<SmallVec<[&str;2]>>();
```
It of course borrow `processed`, but it does not have to borrow still when we use `processed` again.